contentcuration/contentcuration/frontend/administration/views/ChannelActionsDropdown.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+
+  <div>
+    <ConfirmationDialog
+      v-model="restoreDialog"
+      :title="$tr('restoreHeading', {name})"
+      :text="$tr('restoreText', {name})"
+      :confirmButtonText="$tr('restore')"
+      :confirmHandler="restoreHandler"
+    />
+    <ConfirmationDialog
+      v-model="makePublicDialog"
+      :title="$tr('makePublicHeading', {name})"
+      :text="$tr('makePublicText', {name})"
+      :confirmButtonText="$tr('makePublic', {name})"
+      :confirmHandler="makePublicHandler"
+    />
+    <ConfirmationDialog
+      v-model="makePrivateDialog"
+      :title="$tr('makePrivateHeading', {name})"
+      :text="$tr('makePrivateText', {name})"
+      :confirmButtonText="$tr('makePrivate', {name})"
+      :confirmHandler="makePrivateHandler"
+    />
+    <ConfirmationDialog
+      v-model="deleteDialog"
+      :title="$tr('deleteHeading', {name})"
+      :text="$tr('deleteText', {name})"
+      :confirmButtonText="$tr('delete', {name})"
+      :confirmHandler="deleteHandler"
+    />
+    <VMenu>
+      <template v-slot:activator="{ on }">
+        <v-btn
```

Comment:
Please convert to uppercase formatting for consistency across the repo (v-btn -> VBtn)